ckanext/dcatapit/harvesters/csw_harvester.py

```diff
@@ -236,25 +286,64 @@ def get_package_dict(self, iso_values, harvest_object):
 
         package_dict['extras'].append({'key': 'language', 'value': language})
 
-        #  -- temporal_coverage -- #
+        # temporal_coverage
+        # ##################
+        temporal_coverage = []
+        temporal_start = None
+        temporal_end = None
+
         for key in ['temporal-extent-begin', 'temporal-extent-end']:
             if len(iso_values[key]) > 0:
                 temporal_extent_value = iso_values[key][0]
                 if key == 'temporal-extent-begin':
-                    package_dict['extras'].append({'key': 'temporal_start', 'value': temporal_extent_value})
+                    temporal_start = temporal_extent_value
                 if key == 'temporal-extent-end':
-                    package_dict['extras'].append({'key': 'temporal_end', 'value': temporal_extent_value})
-
-        #  -- conforms_to -- #
-        conforms_to = iso_values["conformity-specification-title"]
-        package_dict['extras'].append({'key': 'conforms_to', 'value': conforms_to})
+                    temporal_end = temporal_extent_value
+        if temporal_start:
+            temporal_coverage.append({'temporal_start': temporal_start,
+                                      'temporal_end': temporal_end})
+        if temporal_coverage:
+            package_dict['extras'].append({'key': 'temporal_coverage', 'value': json.dumps(temporal_coverage)})
+
+        # conforms_to
+        # ##################
+        conforms_to_identifier = iso_values["conformity-specification-title"]
+        conforms_to_locale = self._ckan_locales_mapping.get(iso_values["metadata-language"], 'it').lower()
+
+        conforms_to = {'identifier': conforms_to_identifier,
+                       'title': {conforms_to_locale: conforms_to_identifier}}
+
+        for entry in iso_values["conformity-title-text"]:
+            if entry['text'] and entry['locale'].lower()[1:]:
+                conforms_to_locale = self._ckan_locales_mapping[entry['locale'].lower()[1:]]
+                if self._ckan_locales_mapping[entry['locale'].lower()[1:]]:
+                    conforms_to['title'][conforms_to_locale] = entry['text']
+        
+        if conforms_to:
+            package_dict['extras'].append({'key': 'conforms_to', 'value': json.dumps([conforms_to])})
 
-        #  -- creator -- #
+        # creator
+        # ###############
         citedResponsiblePartys = iso_values["cited-responsible-party"]
-        agent_name, agent_code = utils.get_responsible_party(citedResponsiblePartys, \
-            agents.get('author', self._dcatapit_config.get('agents').get('author')))
-        package_dict['extras'].append({'key': 'creator_name', 'value': agent_name})
-        package_dict['extras'].append({'key': 'creator_identifier', 'value': agent_code or default_agent_code})
+        self.localized_creator = []
```

Comment:
As we discussed, we don't need handle localized value for this. Please revert this piece of code.